opnode/rollup/driver/step.go

```diff
@@ -51,9 +51,9 @@ func (d *outputImpl) newBlock(ctx context.Context, l2Finalized eth.BlockID, l2Pa
 		return l2Parent, nil, fmt.Errorf("failed to fetch L1 block info of %s: %v", l1Origin, err)
 	}
 
-	timestamp := l2Info.Time() + d.Config.BlockTime
+	timestamp := l2Info.Time() + d.Config.MaxSequencerTimeDiff
```

Comment:
Shouldn't this still be the block time? This is for the block that is being created rather than verifying that a timestamp is valid.

Reply:
ah right, good catch